### PR TITLE
Feature/2018 08 07 marcus hellberg

### PR DESCRIPTION
**Pull request by tginsberg**

### CODE REVIEW

File: sections/cjug.md [hunk 1]
```diff
@@ -33,24 +33,47 @@ July 12, 2018
...
 
-## Billy Korando Presents: 
-## Welcome to JUnit 5
+## Ask the Spring Team Anything: 
+## A panel with Spring project leads
 
-* Tuesday, July 24 at Peak6
-* Discover the new features in JUnit 5 and how to migrate your test suite
-* Billy is a software consultant with Keyhole Software, and conference speaker
+* Wednesday, August 29 at TBD
```
Comment (bobpaulin) [inline sections/cjug.md]: Not TBD anymore.  At Pivotal Labs

Comment (tginsberg) [reply sections/cjug.md]: Yeah, I'm on it. No idea why this is so screwed up.

File: sections/cjug.md [hunk 2]
```diff
@@ -33,24 +33,47 @@ July 12, 2018
 * Let our mentors help you
 * Talk about problems/shop/tech
 * Nick, Lamar, Todd, Janine (Mentors)
-* Next == Wednesday, July 18 @ GrubHub
-  * Optional topic == Kotlin Koans (led by Todd)
+* Next == Monday, August 20 @ Rocketmiles
```
Comment (bobpaulin) [inline sections/cjug.md]: Has the meetup be posted?

Comment (tginsberg) [reply sections/cjug.md]: No, will do it today.

**Comment by tginsberg** [comment]
Sorry about merge conflicts. No idea why it thought that would conflict.

**Pull request merged**